Configuration/Objects.yaml

```diff
@@ -10,3 +10,70 @@
     2:
       # cache lifetime
       value: 10
+
+'Flowpack\Media\Ui\GraphQL\Middleware\GraphQLMiddlewareFactory':
+  arguments:
+    1:
+      setting: Flowpack.Media.Ui.GraphQL.debugMode
+    2:
+      setting: Flowpack.Media.Ui.GraphQL.corsOrigin
+    3:
+      object:
+        factoryObjectName: 'Neos\Flow\Cache\CacheManager'
+        factoryMethodName: 'getCache'
+        arguments:
+          1:
+            value: 'Flowpack_Media_Ui_GraphQLSchemaCache'
+
+'Flowpack.Media.Ui:GraphQLMiddleware':
+  className: 'Flowpack\Media\Ui\GraphQL\Middleware\GraphQLMiddleware'
+  scope: singleton
+  factoryObjectName: 'Flowpack\Media\Ui\GraphQL\Middleware\GraphQLMiddlewareFactory'
+  arguments:
+    1:
+      # GraphQL URL
+      value: '/neos/graphql/media-assets'
+    2:
+      # PHP Class with the Query/Mutation attributed methods
+      value: 'Flowpack\Media\Ui\GraphQL\MediaApi'
+    3:
+      # Look for classes in the following namespaces when resolving types:
+      value:
+        - 'Flowpack\Media\Ui\GraphQL\Types'
+    4:
+      # Simulate a request to the module controller to check if the current user has access to the requested resource
+      value: 'Flowpack\Media\Ui\Controller\MediaController'
+    5:
+      # Custom resolvers
```

Comment:
i dont understand why the resolvers a custom :D  ? what makes them custom? Is there a native implementation already?